doc/source/tutorial/optimize.rst

```diff
@@ -1534,15 +1534,8 @@ Finally, we can solve the transformed problem using :func:`linprog`.
     >>> x3_bounds = (-3.0, None)
     >>> bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
     >>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, bounds=bounds)
-    >>> print(result)
-         con: array([15.5361242 , 16.61288005])  # may vary
-         fun: -370.2321976308326  # may vary
-     message: 'The algorithm terminated successfully and determined that the problem is infeasible.'
-         nit: 6  # may vary
-       slack: array([ 0.79314989, -1.76308532])  # may vary
-      status: 2
-     success: False
-           x: array([ 6.60059391,  3.97366609, -0.52664076,  1.09007993])  # may vary
+    >>> print(result.message)
+    The problem is infeasible. (HiGHS Status 8: model_status is Infeasible; primal_status is b'At lower/fixed bound')
```

Comment:
This byte-string is resolved in gh-16814

Reply:
```suggestion
    The problem is infeasible. (HiGHS Status 8: model_status is Infeasible; primal_status is At lower/fixed bound)
```